### PR TITLE
Micromamba install

### PR DESCRIPTION
**Pull request by Laxzal**
Added Micromamba installation check

### CODE REVIEW

File: devtools/install_gcn_cpu.sh [hunk 1]
```diff
@@ -1,15 +1,30 @@
...
+    echo "Mamba is installed."
     COMMAND_PKG=mamba
-elif command -v conda >/dev/null 2>&1; then
+# Check if Conda is installed
+elif [ -x "$(command -v conda)" ]; then
+    echo "Conda is installed."
     COMMAND_PKG=conda
 else
-    echo "Error: mamba or conda is not installed. Please download and install mamba or conda - we strongly recommend mamba"
-    exit 1
```
Comment (MichalKesl) [inline devtools/install_gcn_cpu.sh:12]: add exit 1

File: devtools/install_kinbot.sh [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
+# Check if Mamba is installed
+elif [ -x "$(command -v mamba)" ]; then
+    echo "Mamba is installed."
+    COMMAND_PKG=mamba
+# Check if Conda is installed
+elif [ -x "$(command -v conda)" ]; then
+    echo "Conda is installed."
+    COMMAND_PKG=conda
+else
+    echo "Micromamba, Mamba, and Conda are not installed. Please download and install one of them - we strongly recommend Micromamba or Mamba."
```
Comment (MichalKesl) [inline devtools/install_kinbot.sh:16]: add exit 1

File: devtools/install_ob.sh [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+# Check if Mamba is installed
+elif [ -x "$(command -v mamba)" ]; then
+    echo "Mamba is installed."
+    COMMAND_PKG=mamba
+# Check if Conda is installed
+elif [ -x "$(command -v conda)" ]; then
+    echo "Conda is installed."
+    COMMAND_PKG=conda
+else
+    echo "Micromamba, Mamba, and Conda are not installed. Please download and install one of them - we strongly recommend Micromamba or Mamba."
```
Comment (MichalKesl) [inline devtools/install_ob.sh:16]: add exit 1

File: devtools/install_sella.sh [hunk 4]
```diff
@@ -1,42 +1,63 @@
...
+    echo "Mamba is installed."
+    COMMAND_PKG=mamba
+# Check if Conda is installed
+elif [ -x "$(command -v conda)" ]; then
+    echo "Conda is installed."
+    COMMAND_PKG=conda
 else
-  echo "mamba and conda are not installed. Please download and install mamba or conda - we strongly recommend mamba"
-  exit 1
+    echo "Micromamba, Mamba, and Conda are not installed. Please download and install one of them - we strongly recommend Micromamba or Mamba."
```
Comment (MichalKesl) [inline devtools/install_sella.sh:16]: add exit 1

File: devtools/install_torchani.sh [hunk 5]
```diff
@@ -1,16 +1,32 @@
...
     COMMAND_PKG=mamba
-elif command -v conda >/dev/null 2>&1; then
+# Check if Conda is installed
+elif [ -x "$(command -v conda)" ]; then
+    echo "Conda is installed."
     COMMAND_PKG=conda
 else
-    echo "Error: mamba or conda is not installed. Please download and install mamba or conda - we strongly recommend mamba"
-    exit 1
+    echo "Micromamba, Mamba, and Conda are not installed. Please download and install one of them - we strongly recommend Micromamba or Mamba."
```
Comment (MichalKesl) [inline devtools/install_torchani.sh:16]: add exit 1

File: devtools/install_xtb.sh [hunk 6]
```diff
@@ -1,19 +1,40 @@
...
+    echo "Mamba is installed."
+    COMMAND_PKG=mamba
+# Check if Conda is installed
+elif [ -x "$(command -v conda)" ]; then
+    echo "Conda is installed."
+    COMMAND_PKG=conda
 else
-  echo "mamba and conda are not installed. Please download and install mamba or conda - we strongly recommend mamba"
-  exit 1
+    echo "Micromamba, Mamba, and Conda are not installed. Please download and install one of them - we strongly recommend Micromamba or Mamba."
```
Comment (MichalKesl) [inline devtools/install_xtb.sh:16]: add exit 1

**Review by MichalKesl — COMMENTED** [review]
Need to add exit 1 at the commented places.

**Review by MichalKesl — DISMISSED** [review]
Looks good to me

**Review by kfir4444 — APPROVED** [review]
Great work @Laxzal !

**Pull request merged**